CONTRIBUTING.md

```diff
@@ -0,0 +1,70 @@
+# Contribution Guide
+
+jargonLint aims to be accessible to all developers and writers, regardless of prior experience. However, we ask that you follow this guide in your contributions. This helps us support consistency and structure within the repository.
+
+## Simple Fixes
+
+There can be simple typos or edits to only a few lines that you want to correct.
+
+In this case, you can simply press the "edit" option in the upper page and GitHub should guide you through:
+1. Creating a fork
+2. Editing the fork
+3. Opening a pull request
+
+If the GitHub suggestions you should see on the upper area are not satisfactory, and you have limited experience, you can simply open an issue for your fix. 
+
+## Adding New Rules
+
+We prefer to add rules that are comprehensive and widely useful. Therefore, we usually cannot accept opinionated rules that strongly differ between projects.
```

Comment:
I didn't want to add too much since it's also mentioned in the README to the project. I hope this is enough here.

Reply:
👍

Maybe we can add that we expect/want jargonLint to be used with other vale styles guide to complete them. And not to replace them.

I'm unsure where to add this, but I think it's important

Reply:
```suggestion
We prefer to add rules that are comprehensive and widely useful. Therefore, we usually cannot accept opinionated rules that strongly differ between projects. The goal of jargonLint, after all, is to create rules that compliment other style guides.
```

How about this? A simple sentence, but I think it should suffice.

Reply:
compliment is wrong
complement is the one you looked for

Reply:
Accept that, I'm fine with your suggestion